docs/faq/getting_help.md

```diff
@@ -0,0 +1,5 @@
+# FAQ - Getting Help
+
+## What resources or support do you provide to jurisdictions interested in using SEED as part of a benchmarking program?
+
+We typically provide technical support within reason as needed. As the community grows, there are more users able to provide support. It is recommended to use [GitHub issues](https://github.com/SEED-platform/seed/issues) to a) view existing feature requests and issues, and b) communicate with various developers.
```

Comment:
"We typically provide as needed technical support within reason."

Reply:
Went with
> We typically provide technical support as needed and within reason.
Hope that's OK.

Reply:
👍